README.RU.md

```diff
@@ -0,0 +1,198 @@
+# `plotman`: менеджер засева Chia
+[English](README.md) / [Русский](README.RU.md)
+
+Эта утилита служит для управления засевом [Chia](https://github.com/Chia-Network/chia-blockchain). 
+Утилита запускается на сеющей машине и предоставляет следующий функционал:
+
+- Автоматическое создание новых заданий засева, возможно, перекрывающихся ("в шахматном порядке") в нескольких временных каталогах скорость ограничена глобально и для каждого временного каталога.
```

Comment:
You really shouldn't copy punctuation and wording into translation. Happens all over the text below.